.github/workflows/ci.yml

```diff
@@ -1,16 +1,19 @@
-name: Erdapfel CI
+name: Qwant Maps - CI
 
-on: [push, pull_request]
```

Comment:
Don't we want to keep the *pull_request* trigger? I think it will run the CI on a merged version of the branch, which is often equivalent to the branch itself but can eventually catch bugs  that would happen once merged on master.

This is not the first time it comes up [this week](https://github.com/Qwant/mimirsbrunn/pull/16#discussion_r929758900), so I'm starting to wonder how relevant my comment is :sweat_smile: